mkat_fpga_tests/test_cbf.py

```diff
@@ -336,6 +333,91 @@ def print_baselines():
             fengops.feng_eq_set(self.correlator, source_name=inp, new_eq=old_eqs)
             zero_inputs.remove(inp)
             nonzero_inputs.add(inp)
-            #zero_baseline, nonzero_baseline = calc_zero_and_nonzero_baselines(nonzero_inputs)
-            #print_baselines()
-        #print self.correlator.feng_eq_get().items()
+
+    def test_back2back_consistency(self):
+        """1. Check that back-to-back dumps with same input are equal"""
+        test_name = '{}.{}'.format(strclass(self.__class__), self._testMethodName)
+        init_dsim_sources(self.dhost)
+        self.dhost.sine_sources.sin_0.set(frequency=self.expected_fc, scale=0.25)
+
+        for i, freq in enumerate(self.requested_test_freqs):
```

Comment:
I don't think we should be sharing requested_test_freqs between the tests since it is not neccesary to do the whole fine frequency scan for the consistency tests. A hand full of frequencies should be enough.